node/core/approval-voting/src/lib.rs

```diff
@@ -1321,15 +1330,6 @@ async fn handle_approved_ancestor(
 				}
 			}
 		}
-		block_descriptions.push(BlockDescription {
```

Comment:
The placement of this `fn push(...)` caused the flakyness of the sanity check

Reply:
I think I need some rationale why this is ok to be removed from `else if` case.